docs/examples/rag_weaviate.ipynb

```diff
@@ -0,0 +1,1092 @@
+              "\u001b[1;32m╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯\u001b[0m\n"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        }
+      ],
+      "source": [
+        "# Create a prompt where context from the Weaviate collection will be injected\n",
+        "prompt = \"Explain how {text} works, using only the retrieved context.\"\n",
+        "query = \"a generative adversarial net\"\n",
+        "\n",
+        "response = collection.generate.near_text(\n",
+        "    query=query,\n",
+        "    limit=3,\n",
+        "    grouped_task=prompt,\n",
+        "    return_properties=[\"text\", \"title\"]\n",
+        ")\n",
+        "\n",
+        "# Prettify the output using Rich\n",
+        "console = Console()\n",
+        "\n",
+        "console.print(Panel(f\"{prompt}\".replace(\"{text}\", query), title=\"Prompt\", border_style=\"bold red\"))\n",
+        "console.print(Panel(response.generated, title=\"Generated Content\", border_style=\"bold green\"))"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "7tGz49nfUegG"
+      },
+      "source": [
+        "We can see that our RAG pipeline performs relatively well for simple queries, especially given the small size of the dataset. Scaling this method for converting a larger sample of PDFs would require more compute (GPUs) and a more advanced deployment of Weaviate (like Docker, Kubernetes, or Weaviate Cloud). However, this notebook demonstrates that Docling is a robust and powerful open source tool for converting PDFs to structured data.\n",
+        "\n",
+        "To take this solution to next level, you consider:\n",
+        "* Experimenting with different [chunking techniques](https://weaviate.io/developers/academy/py/standalone/chunking/introduction)\n",
+        "* Using a RAG framework like [DSPy](https://weaviate.io/developers/integrations/llm-frameworks/dspy), [LlamaIndex](https://docs.llamaindex.ai/en/stable/api_reference/storage/vector_store/weaviate/), or [LangChain](https://python.langchain.com/docs/integrations/vectorstores/weaviate/)\n",
+        "* Implementing [advanced RAG](https://weaviate.io/blog/advanced-rag) techniques, like [Agentic RAG](https://weaviate.io/blog/what-is-agentic-rag)"
+      ]
+    }
+  ],
+  "metadata": {
+    "accelerator": "GPU",
+    "colab": {
+      "gpuType": "T4",
+      "provenance": []
+    },
+    "kernelspec": {
+      "display_name": "Python 3",
+      "name": "python3"
+    },
+    "language_info": {
+      "codemirror_mode": {
+        "name": "ipython",
+        "version": 3
+      },
+      "file_extension": ".py",
+      "mimetype": "text/x-python",
+      "name": "python",
+      "nbconvert_exporter": "python",
+      "pygments_lexer": "ipython3",
+      "version": "3.11.10"
+    },
+    "widgets": {
```

Comment:
This `metadata.widgets` seems to lead to an issue (see CI "build-docs" failure) when we nbconvert to HTML.

According to https://github.com/jupyter/nbconvert/issues/1731, possible fixes would be:
- [selecting "Widgets" --> "Clear Notebook Widgets State" on Jupyter](https://github.com/jupyter/nbconvert/issues/1731#issuecomment-1292573781)
- [removing the `metadata.widgets` key from the file](https://github.com/jupyter/nbconvert/issues/1731#issuecomment-1113451797)

Reply:
Deleted `metadata.widgets` key from file.